examples/vlp16_params.yaml

```
@@ -0,0 +1,28 @@
+# This is an example filter config for a commercial robot that use Velodyne VLP-16.
+cloud_filter_chain:
+  # Only containment filtering
+  - name: robot_body_filter_containment
+    type: robot_body_filter/RobotBodyFilterPointCloud2
+    params:
+      debug/pcl/clip: false
+      debug/pcl/inside: false
+      debug/pcl/shadow: false
+      sensor/point_by_point: true
+      sensor/min_distance: 0.05
+      sensor/max_distance: 10
+      frames/sensor: lidar_frame
+      frames/filtering: lidar_frame
+      frames/output: lidar_frame
+      frames/fixed: odom # This example is for outdoor, non-mapping application. For more information of how to choose the fixed frame, read https://github.com/peci1/robot_body_filter/pull/18#discussion_r780655811
```

```suggestion
      sensor/min_distance: 0.05
      sensor/max_distance: 10
      # For more information of how to choose the fixed frame and configure point_by_point, see the discussion at https://github.com/peci1/robot_body_filter/pull/18#discussion_r780655811
      sensor/point_by_point: true
      frames/fixed: odom
      frames/sensor: lidar_frame
      frames/filtering: lidar_frame
      frames/output: lidar_frame
```

I suggest reorganizing to get `fixed_frame` and `point_by_point` right next to each other, and referring to both of them in the comment.

---

Modified it as your suggestion. Thanks